common/list/_lists.scss

```diff
@@ -77,3 +75,56 @@ ul.list-col-xs-1 > li { flex-basis: 100% }
 ul.list-col-xs-2 > li { flex-basis: 50% }
 ul.list-col-xs-3 > li { flex-basis: 33.33% }
 ul.list-col-xs-4 > li { flex-basis: 25% }
+
+
+div.lst-lwr-alph>ol,
+.lst-lwr-alph {
+	list-style-type: lower-alpha;
+}
+
+div.lst-upr-alph>ol,
+.lst-upr-alph {
+	list-style-type: upper-alpha;
+}
+
+div.lst-lwr-rmn>ol,
+.lst-lwr-rmn {
+	list-style-type: lower-roman;
+}
+
+div.lst-upr-rmn>ol,
+.lst-upr-rmn {
+	list-style-type: upper-roman;
+}
+
+div.lst-num>ol,
+.lst-num {
+	list-style-type: decimal;
+}
+
+div.lst-none>ul,
+.lst-none {
+	list-style-type: none;
+}
+
+div.lst-spcd>ul>li,
+.lst-spcd>li {
+	margin-bottom: 10px;
+}
+
+div.list-unstyled > ul, .list-unstyled {
+	list-style: none;
+	padding-left: 0;
+}
+
+div.list-inline > ul, .list-inline {
+	list-style: none;
+	margin-left: -5px;
+	padding-left: 0;
+
+	> li {
+		display: inline-block;
+		padding-left: 5px;
+		padding-right: 5px;
+	}
+}
```

Comment:
Where this CSS is coming? I don't see that CSS being removed from the theme.scss? Is it an override? if so, please add a comment that show what file it is overriding or replacing.

Let me know and I will be able to review this piece of code.

Reply:
The code is from theme.min.css and is extended to accomodate the current case.